### ISSUE TITLE
"Test mode" throws an AssertionError

### ISSUE THREAD

**Issue by wagnermaciel** [meteor/tutorials#171]
Hi,

In part 11 of the Blaze simple-todos tutorial I am getting this error

```
W20180517-18:34:00.829(-4)? (STDERR) "[object Object]" reporter blew up with error:
W20180517-18:34:00.852(-4)? (STDERR) AssertionError [ERR_ASSERTION]: path must be a string
```

with this error message

`Exception while invoking method 'mocha/runServerTests' Error: invalid reporter "[object Object]"
`
when attempting to run in "test mode" with

`meteor test --driver-package practicalmeteor:mocha`

**Comment by cur33** [comment]
I had the same problem which I fixed by removing practicalmeteor:mocha (i.e. `meteor remove practicalmeteor:mocha` and then by adding [this fork](https://github.com/cult-of-coders/meteor-mocha/) instead, with the command `meteor add cultofcoders:mocha`. This worked for me. You could also do `meteor add meteortesting:mocha`, but that gave me some errors, so I went back to the cultofcoders package.

**Comment by alewitt** [comment]
im seeing same issue: Meteor 1.7.0.1

**Comment by cur33** [comment]
@alewitt did you attempt my solution?

**Comment by alewitt** [comment]
No. I was just following the tutorial. Its not pressing that i have tests working right now, so i just figured i give it some time to get worked out.

**Comment by tromgy** [comment]
switching to **cultofcoders:mocha** allows tests to run, but the only tests that run are in the **tests/main.js**, not the ones described in the tutorial (**imports/api/tasks.tests.js**)

It doesn't seem to pick up those at all. I even tried to put syntax errors in that file, and there were no error reports. The testing framework seems to ignore it completely.
